run.py

```diff
@@ -98,3 +98,4 @@ def make_shell_context():
 
 if __name__ == "__main__":
     application.run(port=5000)
+# Compulsory task - Setup environment
```

Comment:
Okay will do. Should I create another message for when the user does not exist or just the same one as not background info?

Reply:
I think it's best to create a separate one since we want to encourage logged in user to create their personal background data, so we need to keep the original one.